docs/release/3.5/release-3-5-2.md

```diff
@@ -0,0 +1,117 @@
+OSG Software Release 3.5.2
+===========================
+
+**Release Date:** 2019-10-10
+**Supported OS Versions:** EL7
+
+Summary of changes
+------------------
+
+This release contains:
+
+-   HTCondor-CE 4.0.1
+    -   4.0.0: [Major feature update](https://github.com/htcondor/htcondor-ce/releases/tag/v4.0.0)
+        -   SciTokens support
+        -   Disabled job retries
+        -   configuration reorganization
+    -   4.0.1: [bug fix](https://github.com/htcondor/htcondor-ce/releases/tag/v4.0.1)
+    -   [Some manual adjustments are requiread](/release/release_series#updating-to-htcondor-ce-4x)
```

Comment:
This is very important, I'd maybe add this to the `HTCondor-CE 4.0.1` main bullet above

Reply:
Making this the first bullet